include/ofi_list.h

```diff
@@ -557,6 +557,69 @@ slist_splice_tail(struct slist *dst, struct slist *src)
 	return dst;
 }
 
+/*
+ * Singly-linked list with blocking wait-until-avail support
+ */
+
+struct slistfd {
+	struct slist 		list;
+	struct fd_signal	signal;
+};
```

Comment:
Is this patch in the correct order in commit list?

Reply:
I think it is. Let me update the dates on the commits to show them in correct order. I think Github sometimes takes the commit dates as the ordering criteria instead of the actual order.